tests/mountingOptions/globalProperties.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { h } from 'vue'
+import { mount } from '../../src'
+
+describe('globalProperties', () => {
+  it('adds globally available properties to the component', () => {
+    const Component = {
+      render() {
+        return h('div', this.foo)
+      }
+    }
+    const wrapper = mount(Component, {
+      global: {
+        globalProperties: {
```

Comment:
Is there any use case for to override a global property one just one component?

Reply:
Idk... Data there would most probably be static, or just functions.

Reply:
Sorry I forgot about this. I still don't see the use case for `globalProperties` inside of `global`. Can we remove it (at least until someone has a real use case).

Everything else looks good.

Reply:
You want to remove it entirely, or remove it from `global` but keep it in the global `config` file?
I think it makes sense to have this somewhere, as it is an API, that is available to the App, and people would use it to provide fallbacks for old projects, like filters, constants or attached properties to the `prototype`.
We should be able to use the `mocks` for that, I am just not sure if they function the same, reactivity and all.

Reply:
could we put the feature on hold until someone asks for a legitimate use case for it? I understand what Dobromir suggests, and I'd like people to ask for it – providing fallbacks for old projects might be a slippery slope. Maybe `mocks`  is enough after all.

Reply:
The first one @dobromir-hristov  - I think we should keep the global config - that's something beta supported. I was thinking we remove the `globalProperties` mounting option, since `mocks` can already accomplish this (and it's a standard API from beta).

For the global config, I am happy to wait for someone to ask for it.  I can think of at least two common use cases for global config:

- transition stubs
- `$t` for apps using i18n